app/code/Inchoo/Sample03/Controller/Index/Weight.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\Sample03\Controller\Index;
+
+use Magento\Framework\App\ActionInterface;
+use Magento\Framework\Controller\ResultFactory;
+
+class Weight implements ActionInterface
+{
+    /**
+     * @var ResultFactory
+     */
+    protected $resultFactory;
+
+    /**
+     * Weight constructor.
+     * @param ResultFactory $resultFactory
+     */
+    public function __construct(ResultFactory $resultFactory)
+    {
+        $this->resultFactory = $resultFactory;
+    }
+
+    /**
+     * @return \Magento\Framework\Controller\ResultInterface
+     */
+    public function execute()
+    {
+        /** @var \Magento\Framework\View\Result\Page $resultPage */
+        $resultPage = $this->resultFactory->create(ResultFactory::TYPE_PAGE);
+        $resultPage->getConfig()->getTitle()->set(__('Weight Unit List'));
+
+//        $block = $resultPage->getLayout()->getBlock('content'); zasto je block == false
```

Comment:
čemu ovaj komentar?

Reply:
To nije direktno vezano za task, pokušao sam dohvatiti block pomoću naredbe getBlock('content'), ali vrati mi false pa me zanima zašto.

Reply:
Blokovi se kreiraju tek kasnije pa ga zato u controlleru ne možeš dobiti iz layout-a.